docs/design-docs/multi-DM.md

```diff
@@ -66,3 +70,57 @@ behaviors, namely:
      LockingTransactions.client - to ensure that the lock identifier
      is consistent across all replicas.  This change should be
      straightforward.
+     
+## AtLeastOnceRPC + DHT + ConsensusRSM
+
+### Desired behavior
+
+1. Client creates a new instance or obtains a reference to an existing MicroService.
+2. Client makes a call to an application method in Microservice and gets the result back.
+3. Client DM retries on failure for certain duration.
```

Comment:
nit: Rather change this to "client application does not need to retry on failure (as retries with exponential backoff and timeout are automatically provided by AtLeastOnceRPC)".

Reply:
Current implementation doesn't have exponential back-off(which we need to fix). It is in fact simply retries without even fixed time intervals. Hence, updated without exponential backoff part.